serrat/songs.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
+    <meta charset="UTF-8">
+    <title>Serrat</title>
+</head>
+<body>
+    <div id="nav"></div>
+
+    <main>
+        <article>
+            <section>
+                <h4>La mujer que yo quiero</h4>
+                <p>The song talks about the woman the author wants to be with.</p>
+                <a href="https://www.youtube.com/watch?v=vNT2Xq0jKo8">Click here to listen</a>
+            </section>
+            <section>
+                <h4>Algo personal</h4>
+                <p>The song was written as a criticism of political corrupt leaders.</p>
+                <a href="https://www.youtube.com/watch?v=EXEEql6pnXY">Click here to listen</a>
+            </section>
+            <section>
+                <h4>Mediterráneo</h4>
+                <p>Serrat composes a song to the Mediterranean Sea, linking it to both his own life and the influence
+                    he exerts on the countries he bathes. According to an urban legend Serrat may have composed the song
+                    during his time locked in the Monastery of Montserrat,along with other artists,in protest at the
+                    Burgos Process in the late 1970s.</p>
+                <a href="https://www.youtube.com/watch?v=_w2WOHs9wG4">Click here to listen</a>
```

Comment:
Your links doesn't open in a new window.

Reply:
It was fixed with the addition of the yaml.